events-api/src/modules/events/dto/requests/sort-events.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsOptional, IsString, IsIn } from 'class-validator';
+
+export class SortEventsDto {
+  @IsOptional()
+  @IsString()
+  @IsIn(['eventName', 'eventDate', 'eventCreatedAt'])
+  sort_by: string = 'eventName';
```

Comment:
I'd suggest to place a default or fallback value in a more explicit place. It's too easy to miss it